test/acceptance/features/injectBindingsAsFiles.feature

```diff
@@ -323,3 +323,181 @@ Feature: Bindings get injected as files in application
             """
             mysql
             """
+
+    Scenario: SERVICE_BIDNING_ROOT is not defined twice in the deployment after binding
```

Comment:
we need also scenario when `SERVICE_BINDING_ROOT` is already defined on the app, but then no additional env var is added.

Reply:
IMHO, we do not need a new test - we can add additional assertion to https://github.com/redhat-developer/service-binding-operator/blob/master/test/acceptance/features/injectBindingsAsFiles.feature#L71 because checking for `SERVICE_BINDING_ROOT` duplicate is one aspect of projecting binding as files.

Reply:
+1